nltk/chunk/regexp.py

```diff
@@ -1426,7 +1428,8 @@ def demo():
     print("Demonstration of empty grammar:")
 
     cp = chunk.RegexpParser("")
-    print(chunk.accuracy(cp, conll2000.chunked_sents("test.txt", chunk_types=("NP",))))
+    print(chunk.accuracy(cp, conll2000.chunked_sents(
```

Comment:
Not the biggest deal, but curious why some formatting changes appeared? I kinda liked this one better before, although each is fairly subjective

Reply:
:+1: Longer than 80 characters I guess.